azure-iot-hub-devicesdk/samples/authentication_providers.py

```diff
@@ -27,6 +28,13 @@
 # - Pre-defined environment variables (this is especially useful when running as an Azure IoT Edge module)
 env_auth_provider = from_environment()
 
+# - X509 Certificate Authentication Provider'
+cert_file = "../scripts/certificate.pem"
+key_file = "../scripts/key.pem"
```

Comment:
>"../scripts/key.pem" [](start = 11, length = 20)

Recommend a comment on how to create these files.  I assume you just have to run ../scripts/gen_self_signed_cert.py, but I'd rather know for sure.